R/lithuania.R

```diff
@@ -0,0 +1,448 @@
+#' Lithuania Regional Daily COVID-19 Count Data - County
+#' @inherit get_lithuania_regional_cases_with_level_2
+#'
+#' @description Extracts daily COVID-19 data for Lithuania, stratified by county (apskritis)
+#' and municipality (savivaldybe). Some Lithuanian municipalities share names, there being both a
+#' Vilnius city municipality (m. sav.) and a Vilnius regional municipality (r. sav.)
+#'
+#' Data available at \url{https://opendata.arcgis.com/datasets/d49a63c934be4f65a93b6273785a8449_0}.
+#'
+#' It is loaded and then sanitised.
+#'
+#' @return A data frame of COVID cases by county in Lithuania, ready to be used by \code{get_regional_data()}.
+#'
+#' @seealso [get_lithuania_regional_cases_with_level_2()]
+#' @md
+#' @importFrom dplyr %>% across summarise group_by if_else ungroup mutate
+#' @importFrom tidyselect vars_select_helpers
+get_lithuania_regional_cases_only_level_1 <- function(national_data = FALSE,
+                                                      all_osp_fields = FALSE,
+                                                      death_definition = "of",
+                                                      recovered_definition = "official") {
```

Comment:
Please reduce to <80 char line length if possible. This has not been in place previously so isn't required but it would be nice if possible. Otherwise we need to look at setting a different default in our linting to avoid constant flags.

Reply:
I've tried a fix and will push it shortly. Most of the other `get_xxx_regional_cases_only_level_1` functions don't take params and one of the ones which does has a nice short country name. We are ending up with some long function names, so perhaps a change in the format of these `get_` functions could help for that.

Some of the line length warnings may be unavoidable without extending what's permitted --- I think the URL for the Lithuanian data set is 70 characters long. 

I know that the table in the documentation has some *very* wide lines. It's better for me to be able to write it in Markdown and have some wide lines than to have to switch to writing the Rd more natively.